### PR TITLE
[DEV-7854] Update React Content Renderer to support video block layout options

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png
Comment (e1himself) [inline .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png:1]: Why does it have a broken image?

File: package.json [hunk 1]
```diff
@@ -81,9 +81,10 @@
         "react-dom": "18"
     },
     "dependencies": {
+        "@prezly/content-renderer-react-js": "^0.30.1",
```
Comment (e1himself) [inline package.json:84]: The package is a dependency of itself? :thinking:

Comment (fgyimah) [reply package.json:84]: Oops, that was a mistake let me update :(

File: .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png
Comment (fgyimah) [reply .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png:1]: This is an iframe video without iframe embed code

File: .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png
Comment (e1himself) [inline .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png:1]: Why the image is not aligned with the text?

image

File: .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png
Comment (fgyimah) [reply .loki/reference/chrome_iphone7_Elements_Video_Without_Iframe_Embed_Code.png:1]: For that case it uses the url provided as image src (I'm guessing it wasn't able to connect to the youtube link provided

File: .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png
Comment (fgyimah) [reply .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png:1]: It's using contained layout so image is aligned to the center using `thumbnail_with` and `thumbnail_height`

Comment (e1himself) [reply .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png:1]: I believe the image max-width constraints do not apply here. It should be `width: 100%` to match the iframe insertion dimensions, if it was there.

Comment (fgyimah) [reply .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png:1]: Oh okay sure, will make the update

Comment (fgyimah) [reply .loki/reference/chrome_laptop_Elements_Video_With_Layout_Contained.png:1]: Updated

File: .loki/reference/chrome_iphone7_Elements_Video_Without_Thumbnail_And_Iframe_Embed_Code.png
[the remark this answers is not in the document]
Comment (fgyimah) [reply .loki/reference/chrome_iphone7_Elements_Video_Without_Thumbnail_And_Iframe_Embed_Code.png:1]: Fixed

**Review by e1himself — APPROVED** [review]
_Now_, this looks great! :+1:

**Pull request merged**